### PR TITLE
I18n & l10n

### LINKED ISSUE #19: Add option to disable proxy in config
Add `proxy.enabled` option to disable proxy if it's not needed (like in my case)
I'm already working on it in my fork.

### PR DESCRIPTION
**Pull request by F1uctus**
This PR:

- Adds `I18n` class that gets localized strings from resource bundles
- Adds `proxy.enabled` option to disable proxy if it's not needed (like in my case) (Closes #19)
- Adds mine `codeStyles` config files
- Adds all artifacts (`target` dir, all `.class` & `.jar` files) to `.gitignore` (as they should be posted in https://github.com/Zemllia/RPGram/releases)
- Includes `rlforj-alt` lib to enable pathfinding in future
- Formats all code through IDEA
- Improves `Creature` class logic.

### CODE REVIEW

**Comment by artyom-poptsov** [comment]
You really should not integrate so much changes into one PR.  For example, proxy stuff does not related neither to i18l nor l10n.  The same with "Creature logic". 

Imagine that you go to the college to pass an exam on programming and during the exam you shows your skills in juggling, playing the guitar and making mashed potatoes, and in the end you finally solving your programming task: for you, it'll be really hard to get all the things right, and for a teacher it'll be harder to check your work. 

In short: you'd better to make different PRs for each major change.
